README.md

```diff
@@ -62,7 +64,7 @@ You can find numerous talks on the [nf-core events page](https://nf-co.re/events
 
 ## Quick Start
 
-1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=22.10.1`)
+1. Install [`Nextflow`](https://www.nextflow.io/docs/latest/getstarted.html#installation) (`>=21.10.3`)
```

Comment:
same here

Reply:
I didn't change anything in those lines, I think I suspect I had the readme open before I thought of pulling the dev branch, thanks for catching that!